stablehlo/tests/interpret_pad.mlir

```diff
@@ -0,0 +1,86 @@
+// RUN: stablehlo-interpreter --interpret -split-input-file %s | FileCheck %s
```

Comment:
A test for creating zero dimension size result would be interesting.

Reply:
Per the guidelines, let's not have this test since there is no special handling of empty result tensors needed (we end up iterating through the operand with no action, but that is okay since we don't plan on doing any sort of optimizations at this point).

Reply:
Makes sense!